clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/view/KubernetesClusterProvider.groovy

```diff
@@ -148,6 +147,15 @@ class KubernetesClusterProvider implements ClusterProvider<KubernetesCluster> {
       serverGroup.loadBalancers?.each {
         serverGroup.securityGroups.addAll(securityGroups[it])
       }
+
+      def imageList = new ArrayList()
+      for (def container : serverGroup.deployDescription.containers) {
+        imageList.add(container.imageDescription.repository + ":" + container.imageDescription.tag)
+      }
+      if (!imageList.isEmpty()) {
+        Map buildInfo = [images: imageList]
```

Comment:
Why not just attach an empty image list? Also, an empty image list is cause for concern since an RC shouldn't be deployable without containers.